src/variety/heyawake.js

```diff
@@ -235,7 +235,7 @@ AnsCheck:{
 		"checkFractal@ayeheya",
 		"checkShadeCellCount",
 		"checkCountinuousUnshadeCell",
-		"checkRoomRect"
+		"checkRoomRect@heyawake"
```

Comment:
I suppose this removes the rectangular room check for ayeheya, as well. Would a duplicate line

```suggestion
		"checkRoomRect@ayeheya"
```

do this?